newm-chain-grpc/src/main/proto/newm_chain.proto

```diff
@@ -377,4 +378,14 @@ message LedgerAssetMetadataItem {
   string value = 4;
   int32 nest_level = 5;
   repeated LedgerAssetMetadataItem children = 6;
-}
\ No newline at end of file
+}
+
+message VerifySignDataRequest {
+  string public_key_hex = 1;
+  string signature_hex = 2;
+}
+
+message VerifySignDataResponse {
+  bool verified = 1;
+  string message = 2;
+}
```

Comment:
Should we also return the  stake address that was extracted from sigData so `newm-server` make sure it matches the one the client is claiming?

Reply:
Never mind. The stakeAddress is returned in the message.